fashrevwall/twitter_client/TwitterClient.py

```diff
@@ -29,26 +29,21 @@ def _get_twitter_api(self):
         return tweepy.API(self.auth)
 
 
-    def get_images_by_hashtag(self, hashtag, n):
+    def get_images_by_hashtag(self, hashtag):
         """
-        Receives a string hashtag and returns the list of last n Tweets
+        Receives a string hashtag and returns the list of last 24h Tweets
         containing it.
         """
         tweets = []
-        # Work out date of latest tweet in DB, and query since that date only
-        since = self.get_latest_tweet_date()
-        print "since: " + str(since)
-        if since:
-            previous_query = date.today() - timedelta(1)
-            print previous_query
-            results = tweepy.Cursor(self.api.search, q=hashtag, since=previous_query)
-        else:
-            yesterday = date.today() - timedelta(1)
-            print yesterday
-            results = tweepy.Cursor(self.api.search, q=hashtag, since=yesterday)
+        # Tweepy only allows for queries with day, but no time, so we can only
+        # query since yesteday
+        yesterday = date.today() - timedelta(1)
+        results = tweepy.Cursor(self.api.search, q=hashtag, since=yesterday)
         print "Obtained results, processing..."
-        print results.items()
-        for tweet in results.items():
+        for result in results.items():
+            tweets.append(result)
+        tweets = sorted(tweets,  key=lambda tweet: tweet.created_at)
+        for tweet in tweets:
```

Comment:
For each tweet we are currently doing 2-3 DB queries:
- count number of tweets in DB already
- delete 1 if needed
- add one

Have we tested this with a large number? I am worried it will be slow.

So if we fill the database with 20 and try to add 20 more aka worst case scenario? If this seems fast then try 1000?